src/Nautilus/Nautilus.ts

```diff
@@ -345,5 +347,12 @@ export class Nautilus {
       signer: this.signer
     })
   }
+
+  async computeStop(computeStatusConfig: Omit<StopComputeConfig, 'signer'>) {
```

Comment:
thought(non-blocking):  I think `stopCompute` would sound more natural and easier to understand for the function we expose to the user. Would also be consistent with `StopComputeConfig`.